src/nodes.ts

```diff
@@ -25,8 +22,8 @@ export type TextNode = (NHParser.TextNode) & NodeBase
 // region: TypeGuards
 /* ****************************************************************************************************************** */
 
-export const isTextNode = (node: HtmlNode): node is TextNode => node.nodeType === NodeType.TEXT_NODE;
-export const isCommentNode = (node: HtmlNode): node is CommentNode => node.nodeType === NodeType.COMMENT_NODE;
-export const isElementNode = (node: HtmlNode): node is ElementNode => node.nodeType === NodeType.ELEMENT_NODE;
+export const isTextNode = (node: HtmlNode): node is TextNode => node.nodeType === 3;
```

Comment:
Let's create a `NodeType` object in this package, rather than using number literals, so it's more clear.